roles/workstation/files/xscreensaver-config

```diff
@@ -0,0 +1,256 @@
+# XScreenSaver Preferences File
```

Comment:
Previously it was to force the fiberlamp screensaver, but as we no longer use the environment where that was standard this may be unnecessary.  I'd like to hold it over until we figure out what we are doing with the lower performance workstations.